tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestMinors.g.cs

```diff
@@ -3989,20 +3989,6 @@ public class PackageVersionsLatestMinors
 #if DEFAULT_SAMPLES
                 new object[] { string.Empty },
 #else
-#if NET48
-                new object[] { "2.0.0" },
-                new object[] { "2.1.2" },
-                new object[] { "2.2.10" },
-                new object[] { "3.0.4" },
-                new object[] { "3.1.1" },
-                new object[] { "3.2.2" },
-                new object[] { "3.3.1" },
-                new object[] { "3.4.3" },
-                new object[] { "3.5.2" },
-                new object[] { "3.6.4" },
-                new object[] { "3.7.3" },
-                new object[] { "3.8.3" },
-#endif
```

Comment:
my .NET FX!

Reply:
done in [0db15ca](https://github.com/DataDog/dd-trace-dotnet/pull/7070/commits/0db15ca09c2ba13fb50db81c6ca7d527592886df)